<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SymbolList</class>
 <widget class="QWidget" name="SymbolList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Symbol List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0,0,0,1,0,0">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="comboBox_search_field">
       <item>
        <property name="text">
         <string>Command</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Symbol ID</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="RegexLineEdit" name="lineEdit_search">
       <property name="placeholderText">
        <string>Search...</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_filter">
       <property name="text">
        <string notr="true">Filters</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-filters"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clear_filters">
       <property name="text">
        <string>Clear Filters</string>
       </property>
       <property name="icon">
        <iconset theme="edit-clear"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_sort">
       <item>
        <property name="text">
         <string>Sort by Groups</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sorted Ascending</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sorted Descending</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Found:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_count">
       <property name="text">
        <string notr="true">&lt;count&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_filters" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboBox_case">
        <item>
         <property name="text">
          <string>Case Insensitive</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Case Sensitive</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Regex</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_mode">
        <item>
         <property name="text">
          <string>Math &amp; Textmode</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mathmode</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Textmode</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_packages">
        <property name="text">
         <string>Packages</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_encodings">
        <property name="text">
         <string>Font Encodings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_symmetry">
        <item>
         <property name="text">
          <string>Ignore Symmetry</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Symmetric Symbols</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Asymmetric Symbols</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_grouping">
        <item>
         <property name="text">
          <string>No Grouping</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Group by Package</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Group by Similarity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Group by Symmetry</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Group by Negation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Group by Inside</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Minimimum Group Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_group_min_size">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Maximum Group Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_group_max_size">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="listWidget">
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::Batched</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
       <property name="spacing">
        <number>24</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_command">
         <property name="text">
          <string notr="true">&lt;command&gt;</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QSvgWidget" name="widget_symbol_view" native="true">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2"/>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Symbol ID:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_package_label">
           <property name="text">
            <string>Package:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_package">
           <property name="text">
            <string notr="true">&lt;package&gt;</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Mode:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_mode">
           <property name="text">
            <string notr="true">&lt;mode&gt;</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_fontenc_label">
           <property name="text">
            <string>Font Encoding:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_fontenc">
           <property name="text">
            <string notr="true">&lt;fontenc&gt;</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Similar to:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_similar">
           <property name="text">
            <string notr="true">&lt;similar&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Self-symmetry:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_self_symmetry">
           <property name="text">
            <string>&lt;yes/no&gt;</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Symmetrical to:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="label_other_symmetry">
           <property name="text">
            <string>&lt;others&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_id">
           <property name="text">
            <string notr="true">&lt;symbol id&gt;</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_xelatex_required_label">
           <property name="text">
            <string>Compiler:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_xelatex_required">
           <property name="text">
            <string>XeLaTeX or LuaLaTeX required</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Negation of:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_negation">
           <property name="text">
            <string notr="true">&lt;symbol&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Inside of:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="label_inside_shape">
           <property name="text">
            <string notr="true">&lt;shape&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header>PySide6.QWidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RegexLineEdit</class>
   <extends>QLineEdit</extends>
   <header>handtex.CustomQ.CRegexLineEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
